### ISSUE TITLE
Affichage des distiques (alinéas dans les citations) 

### ISSUE THREAD

**Issue by MathildeVrst** [EcrituresNumeriques/stylo#902]
Dans le cadre d'un prochain dossier à paraître dans Sens Public sur l'Anthologie grecque, nous citerons beaucoup d'épigrammes. La majorité sont des distiques élégiaques, qui ont un affichage "formalisé" (déjà dans le mss. grec d'ailleurs). On les cite, puis on a une alternance d'hexamètres (RAS) et de pentamètres (qui eux doivent être indentés), comme ceci : 

![11_241](https://github.com/EcrituresNumeriques/stylo/assets/89861489/1efe6dfe-30ec-4283-9f9c-b255bcfd69d5)

Quelques idées : 
- un truc facile sorti de votre chapeau (:pray:) 
- appeler le ["verse package"](https://ctan.org/pkg/verse) sur Latex (utilisé par ekdosis) mais sans doute bcp de manipulations pour convenir à toutes les sorties 
- créer une classe d'une façon ou d'une autre
- ... ?

**Comment by ggrossetie** [comment]
Avec AsciiDoc? 😏 

```
[verse,Boileau]
Paul ce grand médecin, l’effroi de son quartier,
  Qui causa plus de maux que la peste et la guerre,
Est curé maintenant, et met les gens en terre :
  Il n’a point changé de métier.
```

![image](https://github.com/EcrituresNumeriques/stylo/assets/333276/4fbd198f-a6e2-47af-b216-3b8d39327e3b)

**Comment by RochDLY** [comment]
Il me semble que c'est plutôt un problème à gérer du côté de Sens Public, non ? @marviro

**Comment by marviro** [comment]
non, c'est une classe qui peut intéresser d'autres revues. La solution asciidoc est pertinente: implémentable dans nos exports? Que fait pandoc? Si on a une classe ensuite il faut adapter les templates. Surtout il faudrait regarder le template html, tex et metopes (avec xslt). Donc dexu problèmes:
1. comment baliser en md 
2. comment utiliser les balises dans l'export (issue pour stylo-export)

**Comment by RochDLY** [comment]
je suis un peu réticent à créer des marqueurs Markdown propres à Stylo : ils ne font pas partie d'une saveur particulière et ne seront pas interprétés dans d'autres environnements. Ce qui veut dire que si on télécharge les sources md depuis Stylo, on est bloqué à moins d'avoir le filtre en local. (en plus, pour détailler ça dans la doc ca risque d'être complexe)

**Comment by marviro** [comment]
Oui, mais on l'a déjà fait, par exemple avec la classe `exergue`. Si on utilise le md pandoc flavoured en gros on sait qu'on aura des span ou des div avec classe. Après il faudra tous les documenter. Donc une syntaxe du type:

```
:::{.verse}
mon vers
:::
```

**Comment by thom4parisot** [comment]
En lien avec #1399 ?

Maintenant qu'on a ce qu'il faut pour du balisage infratextuel.
